sea-query-attr/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+use heck::{ToPascalCase, ToSnakeCase};
+use proc_macro::TokenStream;
+use syn::{parse_macro_input, spanned::Spanned, Data, DataStruct, DeriveInput, Fields, Ident};
+
+extern crate proc_macro;
```

Comment:
You can remove `extern crate` after Rust 2018. in most cases [you won't need to use extern crate anymore](https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html).